mono/mini/Makefile.am.in

```diff
@@ -770,7 +770,7 @@ libmonosgen_2_0_la_LIBADD = libmini.la $(interp_libs_with_mini) $(dbg_libs_with_
 libmonosgen_2_0_la_LDFLAGS = $(libmonoldflags) $(monobin_platform_ldflags) $(CCLDFLAGS)
 libmonosgen_2_0_la_LIBTOOLFLAGS = $(MONO_LIBTOOL_TAG)
 
-noinst_LIBRARIES += libmaintest.a
+#noinst_LIBRARIES += libmaintest.a
```

Comment:
Again, is this related? I know Bernhard added the interpreter regression testing stuff, are we now getting rid of it?

Reply:
Unrelated, will fix.